VisualizationTaskDefinition.json

```diff
@@ -91,119 +36,25 @@
           "value": "123456"
         }
       ],
-      "resourceRequirements": null,
-      "ulimits": [],
-      "dnsServers": [],
-      "mountPoints": [],
-      "workingDirectory": null,
-      "secrets": [],
-      "dockerSecurityOptions": [],
-      "memory": null,
-      "memoryReservation": null,
-      "volumesFrom": [],
-      "stopTimeout": null,
+      "mountPoints": [
+        {
+          "containerPath": "/data",
+          "sourceVolume": "data"
+        }
+      ],
       "image": "ghcr.io/geode-solutions/visualization_geodebackend:develop",
-      "startTimeout": null,
-      "firelensConfiguration": null,
-      "dependsOn": null,
-      "disableNetworking": null,
-      "interactive": null,
-      "healthCheck": null,
-      "essential": true,
-      "links": [],
-      "hostname": null,
-      "extraHosts": [],
-      "pseudoTerminal": null,
-      "user": null,
-      "readonlyRootFilesystem": null,
-      "dockerLabels": {},
-      "systemControls": [],
-      "privileged": null,
       "name": "GeodeBackEnd"
     }
   ],
```

Comment:
Do you need traefik to test?

Reply:
I am going to add traefik on the dev branch and merge when everything is working fine